backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleCommandConfig.java

```diff
@@ -271,8 +271,8 @@ public AnsibleCommandConfig logFileSuffix(String logFileSuffix) {
             ansibleCommand.add("--artifact-dir");
             ansibleCommand.add(artifactsDir);
             ansibleCommand.add("-i");
-            ansibleCommand.add("-vvv");
             ansibleCommand.add(String.valueOf(this.uuid));
+            ansibleCommand.add("-vvv");
```

Comment:
It's an option of ansible-runner main command, so it should probably go to line 265:
```
$ ansible-runner start --help
usage: ansible-runner start [-h] [--process-isolation] [--process-isolation-executable PROCESS_ISOLATION_EXECUTABLE] [--process-isolation-path PROCESS_ISOLATION_PATH]
                            [--process-isolation-hide-paths [PROCESS_ISOLATION_HIDE_PATHS [PROCESS_ISOLATION_HIDE_PATHS ...]]]
                            [--process-isolation-show-paths [PROCESS_ISOLATION_SHOW_PATHS [PROCESS_ISOLATION_SHOW_PATHS ...]]]
                            [--process-isolation-ro-paths [PROCESS_ISOLATION_RO_PATHS [PROCESS_ISOLATION_RO_PATHS ...]]]
                            [--directory-isolation-base-path DIRECTORY_ISOLATION_BASE_PATH] [--resource-profiling] [--resource-profiling-base-cgroup RESOURCE_PROFILING_BASE_CGROUP]
                            [--resource-profiling-cpu-poll-interval RESOURCE_PROFILING_CPU_POLL_INTERVAL]
                            [--resource-profiling-memory-poll-interval RESOURCE_PROFILING_MEMORY_POLL_INTERVAL]
                            [--resource-profiling-pid-poll-interval RESOURCE_PROFILING_PID_POLL_INTERVAL] [--resource-profiling-results-dir RESOURCE_PROFILING_RESULTS_DIR]
                            [--version] [--debug] [--logfile LOGFILE] [-b BINARY] [-i IDENT] [--rotate-artifacts ROTATE_ARTIFACTS] [--artifact-dir ARTIFACT_DIR]
                            [--project-dir PROJECT_DIR] [--inventory INVENTORY] [-j] [--omit-event-data] [--only-failed-event-data] [--omit-env-files] [-q] [-v]
                            [-p PLAYBOOK | -m MODULE | -r ROLE] [--limit LIMIT] [--cmdline CMDLINE] [--hosts HOSTS] [--forks FORKS] [--roles-path ROLES_PATH] [--role-vars ROLE_VARS]
                            [--role-skip-facts] [-a MODULE_ARGS] [--container-image CONTAINER_IMAGE] [--container-volume-mount CONTAINER_VOLUME_MOUNTS]
                            [--container-option CONTAINER_OPTIONS]
                            private_data_dir

positional arguments:
  private_data_dir      base directory containing the ansible-runner metadata (project, inventory, env, etc)

optional arguments:
  -h, --help            show this help message and exit
  --process-isolation   Isolate execution. Two methods are supported: (1) using a container engine (e.g. podman or docker) to execute **Ansible**. (2) using a sandbox (e.g. bwrap)
                        which will by default restrict access to /tmp (default=False)
  --process-isolation-executable PROCESS_ISOLATION_EXECUTABLE
                        Process isolation executable or container engine used to isolate execution. (default=podman)
  --process-isolation-path PROCESS_ISOLATION_PATH
                        path that an isolated playbook run will use for staging. (default=/tmp)
  --process-isolation-hide-paths [PROCESS_ISOLATION_HIDE_PATHS [PROCESS_ISOLATION_HIDE_PATHS ...]]
                        list of paths on the system that should be hidden from the playbook run (default=None)
  --process-isolation-show-paths [PROCESS_ISOLATION_SHOW_PATHS [PROCESS_ISOLATION_SHOW_PATHS ...]]
                        list of paths on the system that should be exposed to the playbook run (default=None)
  --process-isolation-ro-paths [PROCESS_ISOLATION_RO_PATHS [PROCESS_ISOLATION_RO_PATHS ...]]
                        list of paths on the system that should be exposed to the playbook run as read-only (default=None)
  --directory-isolation-base-path DIRECTORY_ISOLATION_BASE_PATH
                        copies the project directory to a location in this directory to prevent multiple simultaneous executions from conflicting (default=None)
  --resource-profiling  Records resource utilization during playbook execution
  --resource-profiling-base-cgroup RESOURCE_PROFILING_BASE_CGROUP
                        Top-level cgroup used to collect information on resource utilization. Defaults to ansible-runner
  --resource-profiling-cpu-poll-interval RESOURCE_PROFILING_CPU_POLL_INTERVAL
                        Interval (in seconds) between CPU polling for determining CPU usage. Defaults to 0.25
  --resource-profiling-memory-poll-interval RESOURCE_PROFILING_MEMORY_POLL_INTERVAL
                        Interval (in seconds) between memory polling for determining memory usage. Defaults to 0.25
  --resource-profiling-pid-poll-interval RESOURCE_PROFILING_PID_POLL_INTERVAL
                        Interval (in seconds) between polling PID count for determining number of processes used. Defaults to 0.25
  --resource-profiling-results-dir RESOURCE_PROFILING_RESULTS_DIR
                        Directory where profiling data files should be saved. Defaults to None (profiling_data folder under private data dir is used in this case).
  --version             show program's version number and exit
  --debug               enable ansible-runner debug output logging (default=False)
  --logfile LOGFILE     log output messages to a file (default=None)
  -p PLAYBOOK, --playbook PLAYBOOK
                        invoke an Ansible playbook from the ansible-runner project (See Ansible Playbook Options below)
  -m MODULE, --module MODULE
                        invoke an Ansible module directly without a playbook (See Ansible Module Options below)
  -r ROLE, --role ROLE  invoke an Ansible role directly without a playbook (See Ansible Role Options below)

Ansible Runner Options:
  configuration options for controlling the ansible-runner runtime environment.

  -b BINARY, --binary BINARY
                        specifies the full path pointing to the Ansible binaries (default=None)
  -i IDENT, --ident IDENT
                        an identifier that will be used when generating the artifacts directory and can be used to uniquely identify a playbook run
                        (default=e6aaaae5-8b4f-4cb5-97c3-79489150f75a)
  --rotate-artifacts ROTATE_ARTIFACTS
                        automatically clean up old artifact directories after a given number have been created (default=0, disabled)
  --artifact-dir ARTIFACT_DIR
                        optional path for the artifact root directory (default=<private_data_dir>/artifacts)
  --project-dir PROJECT_DIR
                        optional path for the location of the playbook content directory (default=<private_data_dir/project)
  --inventory INVENTORY
                        optional path for the location of the inventory content directory (default=<private_data_dir>/inventory)
  -j, --json            output the JSON event structure to stdout instead of Ansible output (default=False)
  --omit-event-data     Omits including extra event data in the callback payloads or the Runner payload data files (status and stdout still included)
  --only-failed-event-data
                        Only adds extra event data for failed tasks in the callback payloads or the Runner payload data files (status and stdout still included for other events)
  --omit-env-files      Add flag to prevent the writing of the env directory
  -q, --quiet           disable all messages sent to stdout/stderr (default=False)
  -v                    increase the verbosity with multiple v's (up to 5) of the ansible-playbook output (default=None)

Ansible Options:
  control the ansible[-playbook] execution environment

  --limit LIMIT         matches Ansible's ```--limit``` parameter to further constrain the inventory to be used (default=None)
  --cmdline CMDLINE     command line options to pass to ansible-playbook at execution time (default=None)
  --hosts HOSTS         define the set of hosts to execute against (default=None) Note: this parameter only works with -m or -r
  --forks FORKS         matches Ansible's ```--forks``` parameter to set the number of concurrent processes (default=None)

Ansible Role Options:
  configuration options for directly executing Ansible roles

  --roles-path ROLES_PATH
                        path used to locate the role to be executed (default=None)
  --role-vars ROLE_VARS
                        set of variables to be passed to the role at run time in the form of 'key1=value1 key2=value2 keyN=valueN'(default=None)
  --role-skip-facts     disable fact collection when the role is executed (default=False)

Ansible Module Options:
  configuration options for directly executing Ansible modules

  -a MODULE_ARGS, --args MODULE_ARGS
                        set of arguments to be passed to the module at run time in the form of 'key1=value1 key2=value2 keyN=valueN'(default=None)

Ansible Container Options:
  configuation options for executing Ansible playbooks

  --container-image CONTAINER_IMAGE
                        Container image to use when running an ansible task
  --container-volume-mount CONTAINER_VOLUME_MOUNTS
                        Bind mounts (in the form 'host_dir:/container_dir)'. Can be used more than once to create multiple bind mounts.
  --container-option CONTAINER_OPTIONS
                        Container options to pass to execution engine. Can be used more than once to send multiple options.

```

Reply:
But OK, if it works, fine then

Reply:
maybe it doesn't i think it was a bad idea to add it blindly. let's drop it

it does behave differently, "ansible-runner -vvv run" vs "ansible-runner run -vvv". the latter prints stuff on stdout